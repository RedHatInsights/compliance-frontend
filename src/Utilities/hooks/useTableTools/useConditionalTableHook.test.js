import { renderHook } from '@testing-library/react-hooks';
import useConditionalTableHook from './useConditionalTableHook';

describe('useConditionalTableHook', () => {
    it('returns a hook on condition', () => {
        const mockHook = jest.fn();
        const { result } = renderHook(() => useConditionalTableHook(true, mockHook, {}));
        expect(mockHook).toHaveBeenCalled();
        expect(result).toMatchSnapshot();
    });
});
