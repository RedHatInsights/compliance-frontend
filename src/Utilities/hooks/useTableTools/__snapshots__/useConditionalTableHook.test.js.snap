// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`useConditionalTableHook returns a hook on condition 1`] = `
Object {
  "all": Array [
    undefined,
  ],
  "current": undefined,
  "error": undefined,
}
`;
